config/v1/types_network.go

```diff
@@ -84,8 +84,10 @@ type ClusterNetworkEntry struct {
 	// The complete block for pod IPs.
 	CIDR string `json:"cidr"`
 
-	// The size (prefix) of block to allocate to each node.
+	// The size (prefix) of block to allocate to each node. If this
+	// field is not used by the plugin, it can be left unset.
 	// +kubebuilder:validation:Minimum=0
+	// +optional
 	HostPrefix uint32 `json:"hostPrefix"`
```

Comment:
IIRC, `+optional` marks it as optional on input, but you also need to mark it "omitempty" to cause it to not be shown in YAML _output_ when it's 0. (Someone correct me if I'm wrong.)

```suggestion
	// +optional
	HostPrefix uint32 `json:"hostPrefix,omitempty"`
```

likewise in  `operator/v1/types_network.go`

Reply:
ack, I'll do this. 

regarding not displaying `0` hostPrefixes; wasn't sure if we should care (https://github.com/openshift/cluster-network-operator/pull/709#issuecomment-658151403), but I am ok either ways.